docker-compose.yml

```diff
@@ -372,4 +373,4 @@ services:
       KSQL_OPTS: -Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
                   -Djavax.net.ssl.trustStorePassword=confluent
                   -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.client.keystore.jks
-                  -Djavax.net.ssl.keyStorePassword=confluent
+                  -Djavax.net.ssl.keyStorePassword=confluent -Dhost.name=ksql-cli
```

Comment:
@gwenshap , would setting `KSQL_HOST_NAME=ksql-cli` work instead?  For consistency...

Reply:
I can try.